modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -56,67 +35,53 @@ Each Serverless cluster has the following limits:
-* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/month
-
-After either the 14 day trial expires or the 100 credits expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial, in case you want to extend it. After that, the data is permanently deleted. To extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
-
-=== Get started
+* Ingress: 0.1 credit per GB of data in written into Redpanda
+* Egress: 0.1 credit per GB of data read out of Redpanda
+* Partitions: 0.003 credit (metered in 10 minute fractions) per partition/hour	
+* Storage: 0.1 credit (metered in 10 minute fractions) per GB stored/month
 
-For the beta release, you'll receive an email to activate your Serverless trial. 
+After either the 14 day trial expires or the 100 credits expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial. After that, the data is permanently deleted. 
 
-. In your Welcome email, click to sign up.
-+
-Redpanda prepares a Redpanda Cloud account for you with a "Hello, World!" Serverless cluster in AWS.
+== Get started
 
-. To log in to Redpanda Cloud, run the following command in your terminal:
-+
-```
-rpk cloud login
-```
+Redpanda prepares an account for you within one minute of starting the trial. 
 
-. Follow the CLI output for next steps.
+Your account includes a `welcome` cluster and a `hello-world` topic in AWS you can explore. 
 
 == Interact with your cluster
 
-Similar to BYOC and Dedicated clusters, you can interact with your Serverless cluster with `rpk` CLI commands or with the Redpanda Cloud UI. You can add team members and grant them access with ACLs on the *Security* page. 
-
-Here are some helpful `rpk` commands:
+Similar to BYOC and Dedicated clusters, you can interact with your Serverless cluster with `rpk` CLI commands or with the Redpanda Cloud UI. For example, you can run `rpk topic consume hello-world` to read the messages in the `hello-world` topic explaining the `rpk` steps to stream data. Here are some helpful `rpk` commands:
 
 * xref:reference:rpk/rpk-cloud/rpk-cloud-login.adoc[`rpk cloud login`]: Use this to log in to Redpanda Cloud or to refresh the session.
 * xref:reference:rpk/rpk-topic.adoc[`rpk topic`]: Use this to manage topics, produce data, and consume data. 
-* xref:reference:rpk/rpk-profile/rpk-profile-print.adoc[`rpk profile print`]: Use this to view `rpk` configuration and see the URL for your Serverless cluster.
+* xref:reference:rpk/rpk-profile/rpk-profile-print.adoc[`rpk profile print`]: Use this to view your `rpk` configuration and see the URL for your Serverless cluster.
```

Comment:
Love the examples here. Wondering if we should put logout too.

Reply:
@vladoschreiner Should we include other rpk commands here, including any troubleshooting info with `rpk cloud auth delete` or `rpk profile delete`?